src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,87 @@
+package duke;
+
+import duke.exception.DukeException;
+
+import java.util.*;
+
+/**
+ * Represents the main class for the Duke application.
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    /**
+     * Initialises Duke with a given file save path.
+     * Runs Duke with empty TaskList if save file is not found.
+     *
+     * @param filePath String representation of the file path
```

Comment:
This is very minor, but I think you can add a full stop at the end of the line.

Reply:
Got it, thanks!